python/ugv_sdk_py/src/scout_robot.cpp

```diff
@@ -136,6 +136,15 @@ void BindScoutRobot(pybind11::module &m) {
         .def("get_robot_state", &ScoutRobot::GetRobotState, "Get the robot state")
         .def("get_actuator_state", &ScoutRobot::GetActuatorState, "Get the actuator state")
         .def("get_common_sensor_state", &ScoutRobot::GetCommonSensorState, "Get the common sensor state");
+
+    // ScoutMiniOmniRobot class
+    py::class_<ScoutMiniOmniRobot, ScoutRobot>(m_scout_robot, "ScoutMiniOmniRobot")
+        .def(py::init<ProtocolVersion>(),
+            py::arg("protocol") = ProtocolVersion::AGX_V2,
+            "Constructor for ScoutMiniOmniRobot with optional protocol")
+        .def("set_omni_motion_command", &ScoutOmniInterface::SetMotionCommand,
```

Comment:
Hi Micheal, can you please update the name to set_motion_command to be consistent with the C++ interface. 
Also, we dont have a omni variant on hand to test. So far it looks ok based on candump output, but if you have one, can you please verify it works.